Decisions.Equifax/PrequalificationOfOne/Response/PreQualificationOfOneResponse.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System.Runtime.Serialization;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using Newtonsoft.Json;
+
+namespace Decisions.Equifax.PreQualificationOfOne.Response
+{   
+        [DataContract]
+        [Writable]
+        public class PreQualificationOfOneResponse
+        {
+            [JsonProperty("status")] 
+            public string Status { get; set; }
+
+            [JsonProperty("consumers")] 
+            public Consumers Consumers { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public partial class Consumers
+        {
+            [JsonProperty("equifaxUSConsumerCreditReport")]
+            public virtual EquifaxUsConsumerCreditReport[] EquifaxUsConsumerCreditReport { get; set; }
+        }
+
+        [DataContract]
```

Comment:
Pretty much everything under line 26 can be removed.

Essentially what is going to happen here, is you'll end up with a Decisions.Equifax.PreQualificationOfOne.Response.EquifaxUsConsumerCreditReport object that's going to show up in Search, but not work with the actual CreditReport step.

Reply:
Yeah, unfortunately as we discussed it does not work